azure/ca/.terraform.lock.hcl

```diff
@@ -0,0 +1,59 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Just a question. This file needs to be committed?

Reply:
Yes, FYI.
https://www.hashicorp.com/blog/terraform-0-14-introduces-a-dependency-lock-file-for-providers
```
The dependency lock file launching with Terraform 0.14 is generated automatically when Terraform init is run. The generated lockfile should be committed into version control systems so that Terraform can guarantee to select exactly the same provider versions on future runs.
```